Documentation update for smoothed safegraph signals (#275)
cmu-delphi/delphi-epidata: docs/api/covidcast-signals/safegraph.md

| `full_time_work_prop` | The fraction of mobile devices that spent more than 6 hours at a location other than their home during the daytime (SafeGraph's `full_time_work_behavior_devices / device_count`) |
| `part_time_work_prop` | The fraction of devices that spent between 3 and 6 hours at a location other than their home during the daytime (SafeGraph's `part_time_work_behavior_devices / device_count`) |
| `median_home_dwell_time` | The median time spent at home for all devices at this location for this time period, in minutes |

> | `completely_home_prop_7dav` | Same as `completely_home_prop` but averaged over a 7 day trailing window. |

Reviewer: ```suggestion
| `completely_home_prop_7dav` | Offers a 7-day trailing window average of the `completely_home_prop`. |
```

Not sure if it could be better worded this way, the suggestion is at the discretion of others. If the rewording changes the meaning or description then please disregard and simply use a hyphen between 7 and day (7-day). If not then perhaps all the others could follow suit.

> | `full_time_work_prop_7dav` | Same as `full_time_work_prop` but averaged over a 7 day trailing window. |

Reviewer: ```suggestion
| `full_time_work_prop_7dav` | Offers a 7-day trailing window average of the`full_time_work_prop`. |
```

> | `part_time_work_prop_7dav` | Same as `part_time_work_prop` but averaged over a 7 day trailing window. |

Reviewer: ```suggestion
| `part_time_work_prop_7dav` | Offers a 7-day trailing window average of the`part_time_work_prop`.|
```

> | `median_home_dwell_time_7dav` | Same as `median_home_dwell_time` but averaged over a 7 day trailing window. |

Reviewer: ```suggestion
| `median_home_dwell_time_7dav` | Offers a 7-day trailing window average of the `median_home_dwell_time`.|
```

observation to the county-level distribution. `n` also serves as the sample
size. The same method is used for aggregation to states.

> Because these signals measure mobility each day, there are strong day-of-week effects:
> weekends have substantially different values than weekdays. Users interested in long-term
> trends, rather than mobility on one specific day, may prefer the `7dav` signals, since
> averaging over the preceding 7 days removes these day-of-week effects.

Reviewer: ```suggestion
SafeGraph's signals measure mobility each day, which causes strong day-of-week effects:
weekends have substantially different values than weekdays. Users interested in long-term
trends, rather than mobility on one specific day, may prefer the `7dav` signals since
averaging over the preceding 7 days removes these day-of-week effects.
```
